corehq/apps/repeaters/models.py

```diff
@@ -433,7 +432,7 @@ def payload_doc(self, repeat_record):
         return None
 
 
-class RepeatRecord(Document, LockableMixIn):
+class RepeatRecord(Document):
```

Comment:
repeatrecords are created in signals and modified in tasks.py. none of the functions on the model use `lock_date` or critical sections. the rest of the occurrences are just reading the record. so i think it's safe to delete